### ISSUE TITLE
Do we need self.text_encoder.eval()

### ISSUE THREAD

**Issue by Hiusam** [nickgkan/beauty_detr#8]
Hi, it seems that in the code you fix the text_encoder by setting the requires_grad of its parameters to false. Do we need self.text_encoder.eval()? We have dropout in the text_encoder.

**Comment by nickgkan** [comment]
Hi, you're technically correct that we need to set the text_encoder in eval mode. In practice though, it won't affect much the result after many training iterations.
